.github/workflows/e2e_test.yml

```diff
@@ -112,9 +112,7 @@ jobs:
           key: e2e-docker-images-${{ matrix.java }}-${{ github.run_id }}-${{ github.run_attempt }}
 
   publish-results:
-    permissions:
-      checks: write
-      pull-requests: write
+    permissions: write-all
```

Comment:
Should we still need this?  Can't we revert to the original?  The less permissive, the better.